files/ko/web/html/index.html

```diff
@@ -1,117 +1,197 @@
+  </dd>
+  <dt>
+    <a href="/ko/docs/Web/HTML/Link_types/preload">rel="preload"로 콘텐츠 미리 불러오기</a>
+  </dt>
+  <dd>
+    {{htmlelement("link")}} 요소, {{htmlattrxref("rel", "link")}} 특성의 <code>preload</code> 값을 사용하면 페이지를
+    불러온 직후 빠르게 사용해야 하는 리소스에 대해 선언적인 가져오기 요청을 {{htmlelement("head")}} 안에 추가할 수
+    있습니다. 이렇게 지정한 리소스는 페이지 불러오기 라이프사이클의 초기, 브라우저의 주 렌더링이 시작하기도 전에
+    시작하고, 페이지의 첫 렌더링이 덜 막혀 성능이 개선됩니다. 이 글에서는 <code>preload</code> 작동 방식에 대한 기본적인
+    지침을 제공합니다.
+  </dd>
 </dl>
-</div>
 
-<div class="section">
-<h2 class="Documentation" id="참고서">참고서</h2>
+<h2 class="Documentation" id="references">참고서</h2>
 
 <dl>
- <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Reference">HTML 참고서</a></dt>
- <dd class="landingPageList">HTML은 <strong>요소</strong>로 이루어져있으며, 각각의 요소는 몇 가지 <strong>특성</strong>을 사용해 수정할 수 있습니다. HTML 문서는 서로 <a href="/ko/docs/Web/HTML/Link_types">링크</a>로 연결되어 있습니다.</dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Element">HTML 요소 참고서</a></dt>
- <dd class="landingPageList">모든 {{glossary("HTML")}} {{glossary("Element", "요소")}}의 목록을 살펴보세요.</dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Attributes">HTML 특성 참고서</a></dt>
- <dd class="landingPageList">HTML 요소는 <strong>특성</strong>을 가집니다. 특성은 요소와 요소의 동작 방식을 다양하게 설정하고 수정하는 일련의 추가 값입니다.</dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Global_attributes">전역 특성</a></dt>
- <dd class="landingPageList">전역 특성은 모든 <a href="/ko/docs/Web/HTML/Element">HTML 요소</a>에 지정할 수 있으며, 심지어 표준에 명시되지 않은 요소에도 지정할 수 있습니다. 이는, 비표준 요소를 사용하는 순간 문서가 HTML5를 준수하지 않음에도 불구하고, 비표준 요소 역시 전역 특성을 허용해야 함을 의미합니다.</dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Inline_elements">인라인 요소</a>와 <a href="/ko/docs/Web/HTML/Block-level_elements">블록 레벨 요소</a></dt>
- <dd class="landingPageList">HTML의 요소는 역사적으로 "블록 레벨" 요소와 "인라인" 요소로 분류됐습니다. 인라인 요소는 콘텐츠의 흐름을 끊지 않고, 요소를 구성하는 태그에 할당된 공간만 차지합니다. 블록 레벨 요소는 부모 요소의 전체 공간을 차지하여 "블록"을 만듭니다.</dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Link_types">링크 유형</a></dt>
- <dd class="landingPageList">HTML에선 두 문서간의 관계를 명시할 때 여러 링크 유형을 사용합니다. 유형을 설정할 수 있는 링크 타입 요소들은 <code>&lt;a&gt;</code>, <code>&lt;area&gt;</code>, 또는 <code>&lt;link&gt;</code>가 있습니다.</dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/Media/Formats">HTML 미디어 유형과 형식 안내서</a></dt>
- <dd class="landingPageList">
- <p>{{HTMLElement("audio")}}와 {{HTMLElement("video")}} 요소를 통해 외부 프로그램 없이 오디오와 비디오 미디어를 재생할 수 있습니다.</p>
- </dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/Guide/HTML/Content_categories">HTML 콘텐츠 카테고리</a></dt>
- <dd class="landingPageList">
- <p>HTML 요소는 공통 특성을 공유하는 요소끼리 묶는 콘텐츠 카테고리 한 가지 이상에 속할 수 있습니다. 콘텐츠 카테고리는 느슨한 관계로 서로 간에 어떤 관계를 형성하지는 않지만, 카테고리에서 공유하는 동작이나 관련 규칙을 정의하고 설명할 때, 특히 복잡한 세부사항을 포함할 때는 더욱 도움이 됩니다.</p>
- </dd>
- <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Quirks_Mode_and_Standards_Mode">호환성 모드와 표준 모드</a></dt>
- <dd class="landingPageList">호환성 모드와 표준 모드에 대한 역사적인 정보.</dd>
+  <dt><a href="/ko/docs/Web/HTML/Reference">HTML 참고서</a></dt>
```

Comment:
```suggestion
  <dt class="landingPageList"><a href="/ko/docs/Web/HTML/Reference">HTML 참고서</a></dt>
```
원본에는 class가 있어서 혹시 삭제하신 이유가 있으신지 궁금해서 리뷰했습니다.

![image](https://user-images.githubusercontent.com/22424891/120212511-0a011900-c26d-11eb-8865-6c2e613bcd93.png)

Reply:
`.landingPageList`는 사용하지 않는 클래스입니다. 과거 2열 레이아웃의 아티팩트이고, 이것도 mdn/content에 올려야 합니다.

Reply:
mdn/content#5520